xml/System.Diagnostics.Metrics/Meter.xml

```diff
@@ -294,13 +296,14 @@ Counter is an instrument that supports non-negative increments. Example uses for
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="name">To be added.</param>
-        <param name="unit">To be added.</param>
-        <param name="description">To be added.</param>
-        <param name="tags">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The instrument name. cannot be <see langword="null" />.</param>
+        <param name="unit">Optional instrument unit of measurements.</param>
+        <param name="description">Optional instrument description.</param>
+        <param name="tags">tags to attach to the counter.</param>
+        <summary>Create a metrics Counter object.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>Counter is an Instrument which supports non-negative increments.
+ Example uses for Counter: count the number of bytes received, count the number of requests completed, count the number of accounts created, count the number of checkpoints run, and count the number of HTTP 5xx errors.</remarks>
```

Comment:
This might be better as a list:
```suggestion
        <remarks>
          <para>Counter is an Instrument that supports non-negative increments.</para>
          <para>Example uses for Counter:</para>
            <ul>
              <li>Count the number of bytes received.</li>
              <li>Count the number of requests completed.</li>
              <li>Count the number of accounts created.</li>
              <li>Count the number of checkpoints run.</li>
              <li>Count the number of HTTP 5xx errors.</li>
            </ul>
          </remarks>
```